tests/bootstrap.php

```diff
@@ -101,6 +101,11 @@
 ]);
 
 Log::config([
+    // 'queries' => [
+    //     'className' => 'Console',
+    //     'stream' => 'php://stderr',
+    //     'scopes' => ['queriesLog']
+    // ],
```

Comment:
Somewhat. I frequently enable query logging and wanted a quick way to enable it.